data-model/SDP-data-model-templates/person_sdp-template.json

```diff
@@ -68,5 +75,8 @@
   ],
   "contributor": [
     "array of strings (CV)"
-  ]
-}
\ No newline at end of file
+  ],
+  "assoc_ms (or assoc_witness)": [
```

Comment:
As noted above under "role" I wonder if there needs to be more than just the object ID here? (And again, just to clarify, this field will be populated by the CMS when a user connects a person record while editing an object, right?)

We may also want to grab the relation that person has with the object and the name forms. Though this may cause problems with updating data. For instance, if we update the attested name in the object recod (correcting a typo, for example), will the CMS be able to determine which item in this field in the corresponding person record needs to be changed? What if we also need to change the role?